znvis/cameras/base_camera.py

```
@@ -0,0 +1,278 @@
+        view_matrix : np.ndarray shape=(4, 4)
+                The view matrix of the camera.
+        """
+        center = box_size / 2
+        max_distance = np.max(box_size) * 1.1
+        eye = center + np.array([0, 0, max_distance])
+        up = np.array([0, 1, 0])
+        return BaseCamera.look_at(center, eye, up)
+
+    @staticmethod
+    def get_minimal_view_matrix(
+        box_size: np.ndarray, renderer_resolution: np.ndarray = np.array([4, 3])
```

Ruff (0.14.0)

Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)